src/styles/utils/mixins.scss

```diff
@@ -12,6 +12,12 @@
   }
 }
 
+@mixin on-mobile {
+  @media (max-width: 640px) {
+    @content;
+  }
+}
+
```

Comment:
There is no need to create on-mobile mixin, you can just write some styles and then change it on-tablet and on-desktop